src/PubSubClient.cpp

```diff
@@ -368,7 +368,7 @@ uint32_t PubSubClient::readPacket(uint8_t* lengthLength) {
 }
 
 boolean PubSubClient::loop_read() {
-    if (_client == nullptr) {
+    if (_client == NULL) {
```

Comment:
Why do this change?
`NULL` is not a pointer type, so the compiler cannot help you here to check for wrong usage.
`nullptr` is a pointer type, so it is an improvement over the old `NULL`

Reply:
Just because otherwise the travis ci build behind the repo doesn't run

Reply:
Hmm that's strange.
What kind of compiler or compiler settings is used?

Reply:
if you click on details behind "continuous-integration/travis-ci/pr — The Travis CI build failed"
you see all the build system info